source/speech/__init__.py

```diff
@@ -689,7 +733,7 @@ def getIndentationSpeech(indentation: str, formatConfig: Dict[str, bool]) -> Spe
 def speak(  # noqa: C901
 		speechSequence: SpeechSequence,
 		symbolLevel: Optional[int] = None,
-		priority: Optional[Spri] = None
+		priority: Spri = Spri.NORMAL

```

Comment:
This looks unrelated.

Reply:
It is slightly, but an intentional change. I found myself trying to work out what the default priority was, and found I could simplify / clarify this.